oam/install_scripts/columnstore-pre-uninstall.in

```diff
@@ -12,6 +12,20 @@ running_systemd() {
     fi
 }
 
+if [[ -f /etc/mysql/debian.cnf ]]; then
+  MDB="/usr/bin/mysql --defaults-file=/etc/mysql/debian.cnf"
+else
+  MDB="/usr/bin/mysql"
+fi
+
+$MDB -e 'DROP DATABASE IF EXISTS infinidb_querystats;' 2>/dev/null
+$MDB -e 'DELETE FROM mysql.func WHERE dl="libcalmysql.so"' 2>/dev/null
+$MDB -e 'DROP DATABASE IF EXISTS calpontsys;' 2>/dev/null
+$MDB -e 'DROP PROCEDURE IF EXISTS infinidb_querystats.calSetUserPriority;' 2>/dev/null
+$MDB -e "DROP PROCEDURE IF EXISTS infinidb_querystats.calRemoveUserPriority(IN host VARCHAR(50), IN usr VARCHAR(50))" 2>/dev/null
+$MDB -e "DROP PROCEDURE IF EXISTS infinidb_querystats.calShowProcessList;"2>/dev/null
+$MDB -e "DROP DATABASE IF EXISTS columnstore_info;" 2>/dev/null
```

Comment:
Just confirming this list is complete.  Do we need to also deregister the engine, anything else?